mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+"""
+General comparison of 2-d model fields against data.  Currently only supports
+mixed layer depths (mld) and sea surface temperature (sst)
+
+Author: Luke Van Roekel, Milena Veneziani
+Last Modified: 10/24/2016
+"""
+
+import matplotlib.pyplot as plt
+import matplotlib.colors as cols
+
+import numpy as np
+import xarray as xr
+import datetime
+from netCDF4 import Dataset as netcdf_dataset
+
+from ..shared.mpas_xarray.mpas_xarray import preprocess_mpas, remove_repeated_time_index
+from ..shared.plot.plotting import plot_global_comparison
+from ..shared.interpolation.interpolate import interp_fields, init_tree
+from ..shared.constants import constants
+
+
+def ocn_modelvsobs(config, field):
```

Comment:
@vanroekel, could we name both the file and the function `modelvsobs` to be consistent with what we're doing in the `seaice` folder?